E2E/tests/avatarUpload.test.ts

```diff
@@ -0,0 +1,78 @@
+Feature('Avatar upload')
+
+const { I, login, game } = inject();
+import username = require("../test_data/usernames");
+import imageLink = require("../test_data/imageLinks");
+import ChessTile = require("../test_data/chessTiles");
+import { ChessPiece } from "../test_data/chessPieces";
+import { ChessPieceValue } from "../test_data/chessPieces";
```

Comment:
`import { ChessPiece, ChessPieceValue } from "../test_data/chessPieces";`

You can import more than one thing from the same file using one line.